### PR TITLE
WIP: More tests

### PR DESCRIPTION
**Pull request by Eric89GXL**
A couple of quick tests were decorated with @expensive_test, and a really slow (~1 min) one wasn't. @dengemann am I missing something?

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Ahh `expensive_test` has to do with file size, not speed. I'll add some comment

File: hcp/io/tests/test_io.py [hunk 1]
```diff
@@ -79,7 +79,6 @@ def test_read_raw_task():
             _basic_raw_checks(raw=raw)
 
 
-@expensive_test
```
Comment (Eric89GXL) [inline hcp/io/tests/test_io.py]: I looked at the `keys` stuff in `tests/config.py` and it looks like the empty-room data should actually be in there. Any idea how to get it to work?

File: hcp/tests/config.py [hunk 2]
```diff
@@ -67,10 +69,32 @@
...
+        if not op.exists(new_file):
+            print('downloading:\n\tfrom %s\n\tto %s' % (s3key, new_path))
+            call(['s3cmd', 'get', s3key, new_path], shell=False)
+            assert op.exists(new_file)
+
 ##############################################################################
 # variable used in different tests
 
 hcp_path = op.expanduser('~/mne-hcp-data/HCP')
+subjects_dir = op.expanduser('~/mne-hcp-data/subjects')
```
Comment (Eric89GXL) [inline hcp/tests/config.py]: @dengemann I have no idea what this should be...? Also, I need to get data from you so I can test locally

**Comment by Eric89GXL** [comment]
Up to 83% we go

@dengemann the next big gains would be from processing some empty room files. Not sure whether or not that's a good idea from an IO/downloading standpoint.

File: hcp/io/tests/test_io.py (hunk 1 above)
Comment (dengemann) [inline hcp/io/tests/test_io.py]: What is the problem? 'noise_empty_room' should be there.

**Comment by Eric89GXL** [comment]
Check out some of the old Travis runs, when I allow it to run the empty-room raw test it fails

**Comment by Eric89GXL** [comment]
...or just try it, maybe you can fix it quickly

**Comment by dengemann** [comment]
it is a disk space I suppose. I think I removed it for this reason.

**Pull request merged**